lit-base/lambda_function.py

```diff
@@ -0,0 +1,24 @@
+from model_service import preprocess, inference, postprocess
+
+
+class BaseResponse:
+
+    def __init__(self, response_body):
+        self.headers = {"content-type": "application/json", "Access-Control-Allow-Origin": "*"}
+        self.statusCode = 200
+        self.body = response_body
+
+    def get_dict(self):
+        return {"headers":self.headers, "body":self.body, "statusCode":self.statusCode}
+
+
+def lambda_handler(event, context):
```

Comment:
Please add comments to **every method/class** on what it is doing ?
follow the format similar to 
https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/executor.py#L113

Reply:
Sure thanks! Also, for BaseResponse->HTTPResponse should we try to avoid using the same name as https://docs.python.org/2.2/lib/httpresponse-objects.html or it is fine?